examples/cpp/helloworld/greeter_async_client.cc

```diff
@@ -23,7 +23,11 @@
 #include <grpcpp/grpcpp.h>
 #include <grpc/support/log.h>
 
+#ifdef BAZEL_BUILD
```

Comment:
Do we really need this?

I've never seen this in other parts of the codebase. Maybe try just keeping line 27? 

Same for the remaining change in other files.

Reply:
I think builds using make would fail if we just keep line 27. This ifdef is used in other examples. For example, see https://github.com/grpc/grpc/blob/master/examples/cpp/helloworld/greeter_client.cc#L25